lib/assembly-objectfile/object_file.rb

```diff
@@ -146,71 +146,13 @@ def mimetype
         check_for_file
         mimetype = ''
         mime_type_order.each do |mime_type_method|
-          mimetype = public_send("#{mime_type_method}_mimetype") if VALID_MIMETYPE_METHODS.include?(mime_type_method)
+          mimetype = send("#{mime_type_method}_mimetype") if VALID_MIMETYPE_METHODS.include?(mime_type_method)
```

Comment:
there is no need to keep the methods public if this is the only consumer of them (including downstream consumers)